content/docs/1.0.x/install/authenticate-cli-bridge/index.md

```diff
@@ -7,113 +7,123 @@ weight: 60
 To authenticate and associate the Keptn CLI and Keptn bridge to your Keptn cluster,
 you need the exposed Keptn endpoint and API token.
 The endpoint values and the method of querying them
-is determined by the access options you use.
+are determined by the access options you use.
 See [Choose access options](../access) for details.
 
-## Authenticate Keptn CLI
+## Get the Keptn endpoint
 
-To authenticate the Keptn CLI against the Keptn cluster, the exposed Keptn endpoint and API token are required. 
+To authenticate the Keptn CLI against the Keptn cluster,
+the exposed Keptn endpoint and API token are required. 
 After [installing Keptn](../helm-install), you already have your Keptn endpoint.
 
-<details><summary>Get API Token and Authenticate Keptn CLI on **Linux / MacOS**</summary>
-<p>
 
-* Set the environment variable `KEPTN_API_TOKEN`:
+Get the Keptn endpoint from the `api-gateway-nginx`.
+If you are using port-forward to expose Keptn,
+your endpoint is `localhost` and the `port` to which you forwarded Keptn
+such as `http://localhost:8080`.)
 
-```console
-KEPTN_API_TOKEN=$(kubectl get secret keptn-api-token -n keptn -ojsonpath={.data.keptn-api-token} | base64 --decode)
-```
+   ```
+   kubectl -n keptn get service api-gateway-nginx
+   ```
 
-* To authenticate the CLI against the Keptn cluster, use the [keptn auth](../../reference/cli/commands/keptn_auth) command:
+   ```
+   NAME                TYPE        CLUSTER-IP    EXTERNAL-IP                  PORT(S)   AGE
+   api-gateway-nginx   ClusterIP   10.117.0.20   <ENDPOINT_OF_API_GATEWAY>    80/TCP    44m
+   ```
 
-```console
-keptn auth --endpoint=$KEPTN_ENDPOINT --api-token=$KEPTN_API_TOKEN
-```
 
-**Note**: If you receive a warning `Using a file-based storage for the key because the password-store seems to be not set up.` this is because a password store could not be found in your environment. In this case, the credentials are stored in `~/.keptn/.password-store` in your home directory.
-</p>
-</details>
+## Get API Token and Authenticate Keptn CLI
 
-<details><summary>Get API Token and Authenticate Keptn CLI on **Windows**</summary>
-<p>
+**On Linux and MacOS**
 
-Please expand the corresponding section matching your CLI tool:
+1. Set the environment variable `KEPTN_API_TOKEN`:
 
-<details><summary>PowerShell</summary>
-<p>
+   ```
+   KEPTN_API_TOKEN=$(kubectl get secret keptn-api-token -n keptn `
+       -ojsonpath={.data.keptn-api-token} | base64 --decode)
+   ```
 
-For the Windows PowerShell, a small script is provided that installs the `PSYaml` module and sets the environment variables.
+1. To authenticate the CLI against the Keptn cluster,
+   use the [keptn auth](../../reference/cli/commands/keptn_auth) command:
 
-* Set the environment variable `KEPTN_ENDPOINT`:
+   ```
+   keptn auth --endpoint=$KEPTN_ENDPOINT --api-token=$KEPTN_API_TOKEN
```

Comment:
You're using the `KEPTN_ENDPOINT` variable here but you're not showing how it is set before that...

Reply:
Ugh.  I now mention it and link to "Choose access option" for instructions about getting the endpoint, depending on the access option being used, and storing it in an environment variable.  It's ugly but I think it is good enough for now.